player.js

```diff
@@ -1,7 +1,7 @@
 /**
  * An media player
  */
-class MediaPlayer 
+class MediaPlayer extends QPCanvas
```

Comment:
Do we plan on using multiple inheritance when MediaMap is implemented?

Reply:
Not sure if you tagged the right line but the **MediaMap** would be in place of the **media** object that the **MediaPlayer** holds as a property.

Reply:
Oh yeah duh lol